packages/artillery/lib/platform/aws-ecs/legacy/run-cluster.js

```diff
@@ -272,8 +273,9 @@ async function tryRunCluster(scriptPath, options, artilleryReporter) {
 
   context.extraSecrets = options.secret || [];
 
-  const idf = customAlphabet('3456789abcdefghjkmnpqrtwxyz');
-  context.testId = `t${idf(4)}_${idf(29)}_${idf(4)}`;
+  const testRunId = process.env.ARTILLERY_TEST_RUN_ID || generateId('t');
+  context.testId = testRunId;
+  global.artillery.testRunId = testRunId;
```

Comment:
This is needed because Fargate is a separate code path to local/Lambda right now.